gradle.properties

```diff
@@ -21,7 +21,7 @@ gatling_plugin_version=3.8.4
 scala_version=2.13.9
 grgit_version=4.1.0
 h2_version=2.1.214
-jacoco_tool_version=0.8.7
+jacoco_tool_version=0.8.8
```

Comment:
We weren't using this property before this PR, as we should have been. Now we are.